cmd/mount/dir.go

```diff
@@ -16,8 +18,8 @@ import (
 )
 
 var (
-	dirLastReadTimes      = make(map[string]time.Time)
-	dirLastReadTimesMutex sync.RWMutex
+	cachedDirs      = make(map[string]*Dir)
```

Comment:
I think the cachedDirs map is probably unnecessary.  The Dir tree has all that info in already and with a ForgetAll method you can recurse calling the ForgetAll method of child directories very easily.